plugins/catalog/src/components/EntitySwitch/conditions.ts

```diff
@@ -14,11 +14,12 @@
  * limitations under the License.
  */
 
-import {
-  ComponentEntity,
-  ResourceEntity,
-  Entity,
-} from '@backstage/catalog-model';
+import { Entity } from '@backstage/catalog-model';
+
+export interface EntityPredicates {
+  kind: string | string[];
```

Comment:
Let's make this one optional too, just to make it future proof when we add more fields that aren't really related to kind.